src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Fdc3StartupAction.cs

```diff
@@ -46,7 +46,7 @@ public async Task InvokeAsync(StartupContext startupContext, Func<Task> next)
             {
                 var appId = (await _appDirectory.GetApp(startupContext.StartRequest.ModuleId)).AppId;
                 var fdc3InstanceId = startupContext.StartRequest.Parameters.FirstOrDefault(parameter => parameter.Key == Fdc3StartupParameters.Fdc3InstanceId).Value ?? Guid.NewGuid().ToString();
-                var channelId = _options.ChannelId ?? "default";
+                var channelId = _options.ChannelId ?? "fdc3.channel.1";
```

Comment:
I don't think we should hardcode the string `fdc3.channel.1` here. Imo this should come from the default channel set - if there's any.

Reply:
what do you mean by that this should come from the default channel set? (the embedded resource? the option that is set by the user through th IOptions<Fdc3DesktopAgentOptions>?)

Reply:
I mean that this identifier is defined in the [.json file](https://github.com/morganstanley/ComposeUI/blob/main/src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/userChannelSet.json#L3). If we change the file then this hardcoded identifier would become invalid.

Reply:
I agree with Zoli - as of now without implementing any more sophisticated features, I'd get the first entry from the current UserChannel list and use that